anitya/config.py

```diff
@@ -66,32 +66,27 @@
     EMAIL_ERRORS=False,
     BLACKLISTED_USERS=[],
     SESSION_PROTECTION="strong",
-    SOCIAL_AUTH_AUTHENTICATION_BACKENDS=(
-        "social_core.backends.fedora.FedoraOpenId",
-        "social_core.backends.gitlab.GitLabOAuth2",
-        "social_core.backends.github.GithubOAuth2",
-        "social_core.backends.google.GoogleOAuth2",
-        "social_core.backends.open_id.OpenIdAuth",
-    ),
-    SOCIAL_AUTH_STORAGE="social_flask_sqlalchemy.models.FlaskStorage",
-    SOCIAL_AUTH_USER_MODEL="anitya.db.models.User",
-    # Force the application to require HTTPS on authentication redirects.
-    SOCIAL_AUTH_REDIRECT_IS_HTTPS=True,
-    SOCIAL_AUTH_LOGIN_URL="/login/",
-    SOCIAL_AUTH_LOGIN_REDIRECT_URL="/",
-    SOCIAL_AUTH_LOGIN_ERROR_URL="/login-error/",
     LIBRARIESIO_PLATFORM_WHITELIST=[],
     DEFAULT_REGEX=r"(?i)%(name)s(?:[-_]?(?:minsrc|src|source))?[-_]([^-/_\s]+?(?:[-_]"
     r"(?:rc|devel|dev|alpha|beta)\d+)?)(?:[-_](?:minsrc|src|source|asc|release))?"
     r"\.(?:tar|t[bglx]z|tbz2|zip)",
-    # Token for GitHub API
-    GITHUB_ACCESS_TOKEN=None,
     SSE_FEED="http://firehose.libraries.io/events",
     CRON_POOL=10,  # Number of workers for check service
     CHECK_TIMEOUT=600,  # Timeout for check service
     # When this number of failed checks is reached,
     # project will be automatically removed, if no version was retrieved yet
     CHECK_ERROR_THRESHOLD=100,
+    # Enabled authentication backends
```

Comment:
This should be also done in the example config and config for vagrant and container setup.